lib/Interface/c_interface.cpp

```diff
@@ -2094,3 +2094,124 @@ int vc_parseMemExpr(VC vc, const char* s, Expr* oquery, Expr* oasserts)
   }
   return 1;
 }
+
+void _vc_useSolver(VC vc, stp::UserDefinedFlags::SATSolvers solver)
+{
+  /* Helper method to encapsulate setting a solver */
+  stp::STP* stp_i = (stp::STP*)vc;
+  stp::STPMgr* b = stp_i->bm;
+  b->UserFlags.solver_to_use = solver;
+}
+
+bool _vc_isUsingSolver(VC vc, stp::UserDefinedFlags::SATSolvers solver)
+{
+  /* Helper method to encapsulate getting a solver */
+  stp::STP* stp_i = (stp::STP*)vc;
+  stp::STPMgr* b = stp_i->bm;
+  return b->UserFlags.solver_to_use == solver;
+}
+
+bool vc_supportsMinisat(__attribute__((unused)) VC vc)
+{
+  return 1;
+}
+
+bool vc_useMinisat(VC vc)
+{
+  _vc_useSolver(vc, stp::UserDefinedFlags::MINISAT_SOLVER);
+  return 1;
+}
+
+bool vc_isUsingMinisat(VC vc)
+{
+  return _vc_isUsingSolver(vc, stp::UserDefinedFlags::MINISAT_SOLVER);
+}
+
+bool vc_supportsSimplifyingMinisat(__attribute__((unused)) VC vc)
+{
+  return 1;
+}
+
+bool vc_useSimplifyingMinisat(VC vc)
+{
+  _vc_useSolver(vc, stp::UserDefinedFlags::SIMPLIFYING_MINISAT_SOLVER);
+  return 1;
+}
+
+bool vc_isUsingSimplifyingMinisat(VC vc)
+{
+  return _vc_isUsingSolver(vc, stp::UserDefinedFlags::SIMPLIFYING_MINISAT_SOLVER);
+}
+
+bool vc_supportsCryptominisat(__attribute__((unused)) VC vc)
+{
+#ifdef USE_CRYPTOMINISAT
+  return 1;
+#else
+  return 0;
+#endif
+}
+
+bool vc_useCryptominisat(
+#ifndef USE_CRYPTOMINISAT
```

Comment:
Oh, this is unfortunate, isn't it? Instead, can we just add some dummy code like

```
vc;  // silence unused warning
return 0;
```

Reply:
Depends which evil you prefer:

```
/home/avj/stp/stp/lib/Interface/c_interface.cpp:2232:7: warning: statement has no effect [-Wunused-value]
     vc;
       ^
```